src/IDRCloudClient.php

```diff
@@ -246,7 +250,8 @@ public static function convert($opt) {
             $endpoint,
             $result,
             $opt[self::KEY_PARAMETERS],
-            (array_key_exists(self::KEY_CONVERSION_TIMEOUT, $opt)) ? $opt[self::KEY_CONVERSION_TIMEOUT] : false
+            (array_key_exists(self::KEY_CONVERSION_TIMEOUT, $opt)) ? $opt[self::KEY_CONVERSION_TIMEOUT] : false,
+            isset($opt[self::KEY_PROGRESS_CALLBACK]) ? $opt[self::KEY_PROGRESS_CALLBACK] : null
```

Comment:
```suggestion
            (array_key_exists(self::KEY_PROGRESS_CALLBACK, $opt)) ? $opt[self::KEY_PROGRESS_CALLBACK] : null
```
(Not sure how this suggestion didn't get included with the review, but adding now)